lib/scrabble.rb

```diff
@@ -10,4 +11,25 @@ def point_values
       "Y"=>4, "Z"=>10
     }
   end
+
+  def score(word)
+    if word == nil
+      0
+    else
+      point_values.values_at(*word.upcase.split(//)).sum
```

Comment:
Woah! This is cool!

Do keep in mind the readability of your code - we want our code to be decipherable by developers at any level - I know some devs that would have to look up the splat operator 😆